src/test/java/net/sf/jabref/logic/util/strings/StringUtilTest.java

```diff
@@ -313,6 +312,9 @@ public void testExpandAuthorInitialsDoNotAddDot() {
         assertEquals("Moore, O.", StringUtil.expandAuthorInitials("Moore, O."));
         assertEquals("Moore, A. O.", StringUtil.expandAuthorInitials("Moore, A. O."));
         assertEquals("Moore, A.-O.", StringUtil.expandAuthorInitials("Moore, A.-O."));
+        assertEquals("MEmre", StringUtil.expandAuthorInitials("MEmre"));
+        assertEquals("{\\'{E}}douard", StringUtil.expandAuthorInitials("{\\'{E}}douard"));
```

Comment:
It might (eventually) make sense to replace expandAuthorInitials with the name formatter. Even though they doesn't do exactly the same thing, if I recall correctly expandAuthorInitials is used in two importers only and it should be OK to possible change the complete formatting of those author fields.